tools/src/main/java/org/apache/kafka/tools/ProducerPerformance.java

```diff
@@ -247,7 +247,7 @@ private static ArgumentParser argParser() {
                 .required(true)
                 .type(Integer.class)
                 .metavar("THROUGHPUT")
-                .help("throttle maximum message throughput to *approximately* THROUGHPUT messages/sec");
+                .help("throttle maximum message throughput to *approximately* THROUGHPUT messages/sec. Set this to -1 to disable throttling.");
```

Comment:
Wouldn't it be more accurate (and perhaps more informative) to say "Set this to a negative number to disable throttling"?

Reply:
That comment is spot on and I debated that with myself when making the change.
My reasoning was to avoid any confusion or potential for error by naming a specific value. While indeed not strictly speaking correct this will definitely work. With "negative number" someone might use -1,232 (for whatever reason) which could cause a parse failure or something similar.

Reply:
I agree, and I was thinking about the other side of the confusion: what would happen if another negative number is used? No strong opinion on my side though.

Reply:
Any other negative number should have the same effect as -1, I really just picked -1 as a proxy value to simplify the help message.
If it is all the same to you, I'd leave it as is, but if you have any preference at all I'm also happy to change :)

Reply:
No, that's fine. Already merged. Thanks! :)